system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -673,8 +673,9 @@
        (echo-util/grant-all-ingest (s/context) provider-id))
 
      (when grant-all-access-control?
-       (echo-util/grant-system-group-permissions-to-all (s/context))
-       (echo-util/grant-provider-group-permissions-to-all (s/context) provider-id)))))
+       (do
```

Comment:
https://clojuredocs.org/clojure.core/when - there's an implicit do so you shouldn't need to make this change.